aggregator/types.go

```diff
@@ -1,308 +0,0 @@
-// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
any particular reasons to not keep the types.go pattern?

Reply:
I feel it's easier to look at the interfaces alongside their implementations, unless there are multiple implementations of the interface, in which case it makes sense to pull out the interface into its own file. Don't think it's a conventional pattern to put all interfaces in one large `types.go` file even though they sometimes aren't closely related.